docs/admin/installation/servers.rst

```diff
@@ -167,6 +179,11 @@ servers:
 
 Select **Done** and press **Enter** to proceed.
 
+.. warning:: The username and password you choose must be used for both the
+             *Application Server* and the *Monitor Server*. These values will
```

Comment:
re: "These values...", the real reason they have to be the same is that we only have one field for password, which is a SecureDrop-limitation, not any technical limitation. Maybe:

"When you deploy SecureDrop from your *Admin Workstation* in a later step, you will only be allowed to enter one password, so it must be identical on both servers."

Reply:
That's a good point, and I like your suggestion. I just pushed a new version with that wording.